src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,14 +6,15 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_TAG = new Prefix("t/");
-    public static final Prefix PREFIX_TITLE = new Prefix("title/");
-    public static final Prefix PREFIX_TIME = new Prefix("time/");
-    public static final Prefix PREFIX_PLACE = new Prefix("place/");
-    public static final Prefix PREFIX_DESCRIPTION = new Prefix("desc/");
-
+    public static final Prefix PREFIX_NAME = new Prefix("-n");
+    public static final Prefix PREFIX_PHONE = new Prefix("-p");
+    public static final Prefix PREFIX_EMAIL = new Prefix("-e");
+    public static final Prefix PREFIX_ADDRESS = new Prefix("-a");
+    public static final Prefix PREFIX_TAG = new Prefix("-t");
+    public static final Prefix PREFIX_TITLE = new Prefix("-title");
```

Comment:
Should we change the following prefixes to a 1 letter prefix?

Reply:
Lets look into it again in the future,
but currently for contacts all is 1 letter prefix,
and for meeting all is 1 word prefix
so it is somewhat standardized for the moment.